scripts/start-release.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+export MEDLEYDIR=`pwd`
+if [ ! -x run-medley ] ; then
+    echo run from MEDLEYDIR
+    exit 1
+fi
+
+tag=$1
+cd ..
+
+tar cfz medley/tmp/medley-$tag.tar.gz \
+    medley/docs/dinfo \
+    medley/greetfiles/SIMPLE-INI*[TM] \
+    medley/run-medley \
+    medley/loadups/lisp.sysout medley/loadups/full.sysout \
+    medley/fonts/displayfonts \
+    medley/loadups/whereis.hash \
+    medley/library/*[A-Za-z] \
+    medley/internal/library/*[A-Za-z]  \
+    medley/lispusers/*[A-Za-z] \
```

Comment:
This doesn't get all the files -- there are 6 that end in a digit, and there's no particular reason to exclude those in the future.